docker/candlepin-base/cp-test

```diff
@@ -195,11 +218,12 @@ OPTIONS:
   -j <version>     use a specific Java version instead of the auto-detected default
   -v               enable verbose/debug output
   -i               internationalization - validate translation files
+  -g               internationalization - gettext to extract strings for translation from source java files
```

Comment:
Also, similarly to the other PR comment, we probably don't have to add this to the cp-test script, since it doesn't have to run in a docker container.